### PR TITLE
Fix handler for log

### PR DESCRIPTION
**Pull request by RidipDe**
**Description of changes:**
Fixed bug caused due to  another PR 

**Testing**
1. Have you successfully run `npm run build:release` locally?
Yes
2. How did you test these changes?
Able to see the logs in Cloudwatch

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: demos/serverless/src/handlers.js [hunk 1]
```diff
@@ -114,8 +114,8 @@ exports.logs = async (event, context) => {
     putLogEventsInput.sequenceToken = uploadSequence;
   }
   const logEvents = [];
-  if (body.logs.length !== 0) {
-    return response(200, 'application/json', JSON.stringify({}));
+  if (body.logs.length === 0) {
```
Comment (simmkyu) [inline demos/serverless/src/handlers.js:117]: You can move this check to line 103

```
 if (!body.logs || !body.logs.length || !body.meetingId || !body.attendeeId || !body.appName) {
```

Note that `!body.logs.length` will be true if the value is `0` so does the `if` condition

File: demos/serverless/src/handlers.js [hunk 2]
```diff
@@ -128,7 +128,7 @@ exports.logs = async (event, context) => {
   }
   putLogEventsInput.logEvents = logEvents;
   await cloudWatchClient.putLogEvents(putLogEventsInput).promise();
-  return response(200, 'application/json', JSON.stringify({}));
+  return response(200, 'application/json', JSON.stringify({success: "Logs written to Cloudwatch"}));
```
Comment (simmkyu) [inline demos/serverless/src/handlers.js:131]: I don't think this change is necessary.

**Comment by simmkyu** [comment]
https://github.com/aws/amazon-chime-sdk-js/pull/349 has a fix for this. Thanks Ridip.

**Comment by RidipDe** [comment]
Will be merged with another PR

**Pull request closed without merging**